index.html

```diff
@@ -3036,272 +3245,114 @@ <h3>Data Schema Serialization</h3>
-            "forms": [...]
-        },
-        "status": {
-            "title" : "Zustand",
-            "titles": {
-                "en": "Status",
-                "de": "Zustand",
-                "ja": "状態",
-                "zh-Hans": "状态",
-                "zh-Hant": "狀態" },
-            "type": "object",
-            ...
-            "forms": [...]
-        }
-    },
-    ...
-}
-</pre>
-
-</section> <!-- end of id="data-schema-serialization-json"-->
-
-</section> <!-- end of id="serialization-json"-->
+  <p>
+  For such <a>TD Context Extensions</a>, the Thing Descriptions use the <code>@context</code>
+  mechanism known from JSON-LD [[?json-ld11]].
+  When using <a>TD Context Extensions</a>,
+  the value of <code>@context</code> of the <a>Class</a> <code>Thing</code>
+  is an Array with additional elements of type <code>anyURI</code> identifying JSON-LD context files
+  or <a>Map</a> containing namespace IRIs as defined in <a href="#thing"></a>.
+  </p>
 
-<section id="content-extension-section">
-<h2>Context Extension</h2>
+  <p>
+    The serialization rules for complex types in <a href="#td-basic-types-mapping"></a>
+    define the serialization of an extended <code>@context</code> name-value pair.
+    A snippet with <a>TD Context Extensions</a> is given below:
+  </p>
 
-    <p>
-    In addition to the standard vocabulary definitions as defined in
-    <a href="#sec-vocabulary-definition"></a>,
-    the Thing Description offers the possibility to add context knowledge
-    from additional namespaces.
-    This mechanism can be used to enrich the content of the Thing Description
-    instances with additional (e.g., domain-specific) semantics.
-    In addition, it can also be used to specify some configurations
-    and behaviors of the underlying communication protocols announced in the
-    <a>Protocol Bindings</a> serialized in <code>forms</code>.
-    </p>
-    <p>
-    For this extension the Thing Description uses the <code>@context</code>
-    mechanism known from JSON-LD [[?json-ld11]].
-    <span class="rfc2119-assertion" id="td-additional-contexts">
-    When a Thing Description uses multiple contexts,
-    the <code>@context</code> member of the TD root object MUST be an array
-    with the TD context URI followed by an object containing
-    the additional namespaces identified by their prefixes as JSON names.
-    </span>
-    </p>
 <pre class="example">
 {
     "@context": [
         "https://www.w3.org/2019/td/v1",
         {
             "iot": "http://example.org/iot",
             "cov": "http://www.example.org/coap-binding#"
-        }
+        },
+        "https://schema.org/"
     ],
     ...
 }
 </pre>
 
-<section>
+<section class="informative">
 <h3>Semantic Annotations</h3>
 
     <p>
-    The context extension allows to add additional vocabulary to a Thing Description instance.
+    <a>TD Context Extensions</a> allow to add additional vocabulary to a Thing Description instance.
```

Comment:
Grammar fix: "allow the addition of".   Also, should it be "vocabulary terms"?   Not sure about the latter, it depends on what the convention is (now).

Reply:
allow for additional <a>Vocabulary Terms</a> to a Thing Description instance.